tool/autoqa/schemaorg/manual-annotations.js

```diff
@@ -479,12 +486,21 @@ const ENUM_VALUE_NORMALIZE = {
     }
 };
 
+const WHITELISTED_PROPERTIES_BY_DOMAIN = {
```

Comment:
It's low on the list of things to worry about, but we should try to move away from blacklist/whitelist as terminology. It is not a great choice of words, on many levels.